test/rules/accessibility/containersAccessibility.spec.js

```diff
@@ -0,0 +1,74 @@
+        expect(messages[0].message).to.equal('div with class container should have role "tabpanel"');
+        expect(messages[0].rule.id).to.be(ruldId);
+    });
+    it('div with class "container" with role "tabpanel" should not result in an error', function () {
+        var code = `<div class="container" role="tabpanel"  data-bind="attr:{'id':'tabpanel_+'name(),'aria-labeledby':'tab_'+name()}"></span>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(0);
+    });
+
+    it('div with class "container" with no data-bind attribute should result in an error', function () {
+        var code = `<div class="container" role="tabpanel" ></span>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(2);
+        expect(messages[0].message).to.equal('div with class container should have attr binding of id that starts with "tabpanel_"');
+        expect(messages[0].rule.id).to.be(ruldId);
+        expect(messages[1].message).to.equal('div with class container should have attr binding of aria-labeledby to an id that starts with "tab_"');
+        expect(messages[1].rule.id).to.be(ruldId);
+    });
+
+    it('div with class "container" with no attr bind of "id" should result in an error', function () {
+        var code = `<div class="container" role="tabpanel" data-bind="attr:{'aria-labeledby':'tab_'+name()}"></span>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(1);
+        expect(messages[0].message).to.equal('div with class container should have attr binding of id that starts with "tabpanel_"');
+        expect(messages[0].rule.id).to.be(ruldId);
+    });
+
+    it('div with class "container" with attr bind of "id" that does not start with "tabpanel_"should result in an error', function () {
+        var code = `<div class="container" role="tabpanel" data-bind="attr:{'id':name(),aria-labeledby':'tab_'+name()}"></span>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(1);
+        expect(messages[0].message).to.equal('div with class container should have attr binding of id that starts with "tabpanel_"');
+        expect(messages[0].rule.id).to.be(ruldId);
+    });
+
+    it('div with class "container" with no attr bind of "aria-labeledby" should result in an error', function () {
+        var code = `<div class="container" role="tabpanel" data-bind="attr:{'id':'tabpanel_+'name()}"></span>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(1);
+        expect(messages[0].message).to.equal('div with class container should have attr binding of aria-labeledby to an id that starts with "tab_"');
+        expect(messages[0].rule.id).to.be(ruldId);
+    });
+
+    it('div with class "container" with attr bind of "aria-labeledby" that does not start with "tab_"should result in an error', function () {
```

Comment:
wouldn't you have a false negative if another data-bind started with "tab_" ?
for example if the html was:
var code = `<div class="container" role="tabpanel" data-bind="attr:{'someAttribute':'tab_1', 'id':'tabpanel_+'name(),aria-labeledby':name()}"></span>`;
(this situation applies to other tests as well).